pkg/etcdcli/etcdcli.go

```diff
@@ -247,7 +247,7 @@ func (g *etcdClientGetter) GetMember(name string) (*etcdserverpb.Member, error)
 		return nil, err
 	}
 	for _, m := range members {
-		if m.Name == name {
+		if strings.Contains(m.Name, name) {
```

Comment:
this is the wrong fix.  Have a `FuzzyMatchDeprecated43_memberName` or some such method. Don't break this method.